src/Movement/Kinematics/HangprinterKinematics.cpp

```diff
@@ -229,6 +230,8 @@ bool HangprinterKinematics::Configure(unsigned int mCode, GCodeBuffer& gb, const
 	}
 	else if (mCode == 666)
 	{
+		// 0=None, 1=last-top, 2=all-top, 3-half-top, etc
+		gb.TryGetUIValue('T', (uint32_t&)anchorMode, seen);
```

Comment:
This gives a warning. Can I get an example for enum UI somewhere else?

Reply:
Try using an enum class. They let you specify the underlying type. So maybe they let you get away with fewer warnings?

Reply:
I was using the enum class. I solved it with a loval unsigned but there's probably a better solution.
Is there some sort of TryGetEnum or something?
cc @dc42 
I'm still marking this conversation as resolved since the is outdated. Please, look at the latest commit and propose alternatives there.